bench/tests/test_init.py

```diff
@@ -57,11 +57,10 @@ def test_multiple_benches(self):
 		})
 
 	def test_new_site(self):
+		self.init_bench('test-bench')
 		self.new_site("test-site-1.dev")
 
 	def new_site(self, site_name):
-		self.test_init()
```

Comment:
@anandpdoshi 

Previously each call to `new_site()` would call `test_init()` to create new bench and then create a new site. 

In case of `test_drop_site` I was trying to test both scenarios in a single test case(**with and without** ***archived_site_path***), and so `tearDown()` never happened which resulted into failure even after correcting the varibale name. And so, I moved call to `test_init()` to caller function, to avoid creation of bench for these two scenarios.

Also, I replaced it with `init_bench()` as we were executing `test_init()` for all the tests to create a new bench.